content/applications/inventory_and_mrp/inventory/management/misc/batch_transfers.rst

```diff
@@ -1,124 +1,218 @@
+Batch Transfers` page. Click the :guilabel:`New` button to begin creating a batch transfer.
 
-.. image:: batch_transfers/batch_transfers_04.png
-    :align: center
-    :alt: View of the list of transfers to choose for a single batch transfer and how to add them
-          to the batch transfer
+On the batch transfer form, fill the following fields out accordingly:
 
-To see the products to pick for the different transfers, click on
-*Select*. If *Multi-locations* has been activated, the document also shows the locations they have
-been reserved from.
+- :guilabel:`Responsible`: employee assigned to the picking. Leave this field blank if any worker
+  can fulfill this picking.
+- :guilabel:`Operation Type`: select the operation type the picking is categorized under from the
+  drop-down menu.
+- The :guilabel:`Scheduled Date` at the top of the form specifies the date the
+  :guilabel:`Responsible` person should complete the transfer to the output location by.
 
-.. image:: batch_transfers/batch_transfers_05.png
-    :align: center
-    :alt: View of a batch transfer list. Products to pick with their source and target locations
+Next, in the :guilabel:`Transfers` list, click :guilabel:`Add a line` to open the :guilabel:`Add:
+Transfers` window. If the :guilabel:`Operation Type` field was filled, the list will filter transfer
+records matching the selected :guilabel:`Operation Type`. Click the :guilabel:`New` button to create
+a new transfer. Once the transfer records are selected, click :guilabel:`Confirm` to confirm the
+batch picking.
 
-Create a Batch Transfer from the Transfers List View
-====================================================
+.. note::
+   For reviewers; Remove this before publishing (:
+   Create a SO and confirm it after setting the Outgoing shipment option to 2 or 3 steps to generate
+   a Pick entry for the example in runbot. This is because if the runbot was originally configured
+   in 1-step, there would be no entries in the newly created operation type, :guilabel:`Pick`!
+
+.. example::
 
-From the *Transfers List View*, select transfers that should be
-included in the Batch. Then, select *Add to batch* from the *Action*
-list.
+   A new batch transfer assigned to the :guilabel:`Responsible`, `Joel Willis` for the `Pick`
+   :guilabel:`Operation Type`. The :guilabel:`Scheduled Date` is set to August 11.
 
-.. image:: batch_transfers/batch_transfers_06.png
-    :align: center
-    :alt: View of the process to add transfers to a batch transfer from the transfers list view
+    .. image:: batch_transfers/batch-transfer-form.png
+        :align: center
+        :alt: View of *Batch Transfers* form.
 
-Next, determine if you want to add the transfers to an existing draft
-batch transfer or create a new one.
+    Clicking the :guilabel:`Add a line` button opens the :guilabel:`Add:Transfers` window,
+    displaying only picks because :guilabel:`Operation Type` was set to `Pick` on the batch transfer
```

Comment:
```suggestion
    displaying only pickings because the :guilabel:`Operation Type` was set to `Pick` on the batch transfer
```
Either that or "picking operations"